src/CLI/actioner/sonic-cli-neighbors.py

```diff
@@ -25,35 +24,50 @@
 import urllib3
 urllib3.disable_warnings()
 
+#For interface to VRF mapping
+vrfDict = {}
```

Comment:
Clear the globals after use so that when we migrate to clish_pyobj, we dont have cached data from previous requests.